DNN Platform/Library/Entities/Urls/UrlRewriterUtils.cs

```diff
@@ -111,9 +111,16 @@ public static void Log404(HttpRequest request, FriendlyUrlSettings settings, Url
 
             if (request != null)
             {
-                if (request.UrlReferrer != null)
+                try
                 {
-                    log.LogProperties.Add(new LogDetailInfo("Referer", request.UrlReferrer.AbsoluteUri));
+                    if (request.UrlReferrer != null)
+                    {
+                        log.LogProperties.Add(new LogDetailInfo("Referer", request.UrlReferrer.AbsoluteUri));
+                    }
+                }
+                catch (Exception)
```

Comment:
Can this be changed to catch the specific exception type, please?

Reply:
Done